src/runtime/CMakeLists.txt

```
@@ -177,6 +177,7 @@ set(RUNTIME_CXX_FLAGS
     -Wsign-compare
     -Wimplicit-fallthrough
     -Wno-sync-alignment
+    -isystem "${Halide_SOURCE_DIR}/dependencies/vulkan/include"
```

Near the top of this file, we should `find_package` the Vulkan headers, like so:

```cmake
find_package(
    VulkanHeaders 1.3.296 REQUIRED
    HINTS "${Halide_SOURCE_DIR}/dependencies/vulkan"
)
get_target_property(VulkanHeaders_INCLUDE_DIR Vulkan::Headers INTERFACE_INCLUDE_DIRECTORIES)
```

Then here, we would write

```suggestion
    -isystem "${VulkanHeaders_INCLUDE_DIR}"
```

This way, if the user has the Vulkan runtime and headers installed on their system, they can use them. If that header/runtime combo is too old for Halide, they'll know at configure time, rather than getting a runtime error in a pipeline.